tests/test_routing.py

```diff
@@ -15,7 +15,7 @@ def users(request):
 
 
 def user(request):
-    content = "User " + request.path_params["username"]
+    content = "User " + request.path_params["name"]
```

Comment:
Let's leave this as `username` throughout - since it makes more sense in this context.